### DISCUSSION TITLE
Simple editor

### DISCUSSION THREAD

**Discussion by CptHowdy2012** [immich-app/immich#12277] [General]
Some pictures are rotated wrong, it would be nice to have a simple editor function of rotating the image.

**Comment by scrampker** [comment]
Rotate is really I need 99% of the time. Crop would be nice, but not necessary. I find to be fairly frustrating overall. Incorrectly-rotated images are very common. I think a flag just needs to be changed in the file. Nothing huge.

**Comment by YarosMallorca** [comment]
Yes please, this would be very handy!! On both web and mobile.

**Comment by Aggaar** [comment]
Just a editor like the "Iris gallery" would be perfect, simple and effective

<img width="200" alt="Screenshot_20260924_211029_Iris Gallery" src="https://github.com/user-attachments/assets/110b8fba-e7ad-4dea-b2ab-164a1784a3d2" />
<img width="200"  alt="Screenshot_20260924_210635_Iris Gallery" src="https://github.com/user-attachments/assets/7bfb24cf-6672-45ee-a0bc-8fd941e385c7" />
<img width="200"  alt="Screenshot_20260924_210906_Iris Gallery" src="https://github.com/user-attachments/assets/fafd99d1-e40c-465e-8e8f-4ec1db604e74" />
<img width="200"  alt="Screenshot_20260924_211022_Iris Gallery" src="https://github.com/user-attachments/assets/9bfe7fa3-452b-40d6-bb40-08fe3f9e5221" />
